ui/src/pages/frontsite/components/Document/DocumentRelations.js

```diff
@@ -87,10 +88,16 @@ export class DocumentRelations extends Component {
       return (
         <>
           <Divider horizontal>Related</Divider>
-          {this.renderMultiparts()}
-          {this.renderSerials()}
-          {this.renderLanguages()}
-          {this.renderEditions()}
+          {isEmpty(this.relations) ? (
```

Comment:
can you please remove this modification? it is already fixed in my PR fro both Document and Series :)

Reply:
See here: https://github.com/inveniosoftware/invenio-app-ils/pull/789 (screenshot)